syntax = "proto3";

package quran;

message AyahId {
    string id = 1;
}